src/modules/strapiPatch.ts

```diff
@@ -17,6 +17,7 @@ export function strapiPatchSingle(strapiGame: StrapiGame): Game {
 			: [],
 		rules: Array.isArray(strapiGame.rules) ? strapiGame.rules.map((a) => a.description) : [],
 		variation: gameVariation(strapiGame.variation),
+		highlighted: typeof strapiGame.highlighted === 'boolean' ? strapiGame.highlighted : '',
```

Comment:
De default van deze zou `false` moeten zijn ipv `" "`

Reply:
Is waar. Top